compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -2901,7 +2889,7 @@ public BLangNode transform(XMLTextNode xmlTextNode) {
 
     private BLangNode createXMLLiteral(TemplateExpressionNode expressionNode) {
         BLangXMLTextLiteral xmlTextLiteral = (BLangXMLTextLiteral) TreeBuilder.createXMLTextLiteralNode();
-        xmlTextLiteral.pos = getPosition(expressionNode);
+        xmlTextLiteral.pos = getPosition(expressionNode.content().get(0));
```

Comment:
As discussed in the meeting, positions in the new tree should be correct.
Let's verify that and update the test case  with the correct line/column number info.

Reply:
ah alright